# Clears joints buffer
---
bool success
string msg
